products/shopware.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Shopware
+category: server-app
+tags: php-runtime
+iconSlug: shopware
+permalink: /shopware
+versionCommand: ./bin/console -V
+releasePolicyLink: https://www.shopware.com/en/news/shopware-6-versioning-strategy/
+changelogTemplate: "https://www.shopware.com/en/changelog/#release-{{'__LATEST__' | replace:'.','-'}}"
+releaseLabel: "{{'__RELEASE_CYCLE__' | replace:'.','-'}}"
+releaseColumn: true
+releaseDateColumn: true
+releases:
+-   releaseCycle: "6.5"
+    releaseDate: 2023-05-03
+    eol: false
+    latest: "6.5.5.1"
+    latestReleaseDate: 2023-09-05
+
+-   releaseCycle: "6.4"
+    eol: 2023-05-03
+    releaseDate: 2021-05-04
+    latest: "6.4.19.0"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "6.3"
+    eol: 2021-05-04
+    releaseDate: 2020-08-03
+    latest: "6.3.5.4"
+    latestReleaseDate: 2021-04-19
+
+-   releaseCycle: "6.2"
+    eol: 2020-08-03
+    releaseDate: 2020-05-09
+    latest: "6.2.3"
+    latestReleaseDate: 2020-07-13
+
+-   releaseCycle: "6.1"
+    eol: 2020-03-09
+    releaseDate: 2020-01-15
+    latest: "6.1.6"
+    latestReleaseDate: 2020-05-14
+
+-   releaseCycle: "5"
+    lts: true
```

Comment:
```suggestion
```

A version cannot be marked LTS if there is no LTS policy. Is there one ?

CC @usta